test/jdk/sun/java2d/DirectX/OnScreenRenderingResizeTest/OnScreenRenderingResizeTest.java

```diff
@@ -25,25 +25,20 @@
  * @key headful
  * @bug 6664068 6666931
  * @summary Tests that resizing a window to which a tight loop is rendering
- * doesn't produce artifacts or crashes
- * @author Dmitri.Trembovetski@sun.com: area=Graphics
+ *          doesn't produce artifacts or crashes
  * @run main/othervm OnScreenRenderingResizeTest
- * @run main/othervm -Dsun.java2d.d3d=false OnScreenRenderingResizeTest
```

Comment:
DirectDraw/GDI is default pipeline in Windows(https://docs.oracle.com/javase/10/troubleshoot/java-2d-pipeline-rendering-and-properties.htm#JSTGD428 ) and explicitly mentioning d3d=false should not make any difference that's why i have removed it.

But i have verified updated test even with d3d=false and there are no issues. If needed i can update the test to run with d3d=false also.

Reply:
Hi Phil,

I was also not sure about it and tested with d3d=false, thanks for the clarification.
I have re-enabled DirectDraw/GDI run.

Even the Java SE 19 doc says DirectDraw/GDI is default on Windows : https://docs.oracle.com/en/java/javase/19/troubleshoot/java-2d-pipeline-rendering-and-properties.html#GUID-AAB8CC86-652B-4A78-83EB-CDC3F5677A48.

Thanks,
Jay